### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by camelwater**
## Description

- Fix Lounge update JSON data bug
   - The "tier" value needs to begin with the word "Tier"
- fix `/changeraceorder` so that race edits follow race order changes
- MII_CACHE fix? (not sure if this is a contribution to memory issues, but seems like it holds on to unnecessary records that could be removed)
- There's also an issue with the backup picture generation - html2image isn't downloading the file to the right location, and I've no idea where the files could be downloading to

## Checklist

- [x] If code changes were made, they have been tested
- [ ] This PR fixes an issue
- [ ] This PR is not a code change (ie. documentation, typehinting, comments, etc.)

### CODE REVIEW

File: api/css/full_scores_orange_discord.css [hunk 1]
```diff
@@ -42,6 +31,16 @@ body {
 .player_total_wrapper,
 .player_name_wrapper,
 .score_wrapper{
-    font-size: 25px;
```
Comment (BadWolf1023) [inline api/css/full_scores_orange_discord.css:45]: See previous comment

File: MogiUpdate.py [hunk 2]
```diff
@@ -619,7 +619,9 @@ async def textInputUpdate(tableText:str, tier:str, races_played=12, warFormat=No
     
     if tier == "squadqueue":
         tier = determine_tier(id_mapping, is_rt)
-    
+    else:
+        tier = "Tier " + tier
```
Comment (BadWolf1023) [inline MogiUpdate.py:623]: Presumably 255MP changed the API again and it broke TB?

Comment (camelwater) [reply MogiUpdate.py:623]: Yep

File: api/css/full_scores_base.css [hunk 3]
```diff
@@ -14,8 +14,8 @@ body{
     display: flex;
     justify-content: center;
     align-items: center;
-    font-size: 35px;
```
[the remark this answers is not in the document]
Comment (camelwater) [reply api/css/full_scores_base.css:17]: Was testing and I think the height change is better; the font change isn't really a big difference. Don't really know

File: commands.py [hunk 4]
```diff
@@ -144,11 +144,12 @@ def lower_args(args: List[str]) -> List[str]:
     return [arg.lower() for arg in args]
 
 async def download_table_picture(message, table_sorted_data: Dict, image_url: str, table_image_path: str):
-    image_download_success = await common.download_image(image_url, table_image_path)
```
[the remark this answers is not in the document]
Comment (camelwater) [reply commands.py:147]: Mistake

[the remark this answers is not in the document]
Comment (camelwater) [reply commands.py:147]: I was testing the backup table generation, which doesn't work on the server. It works on my PC though. I think it has something to do with html2image not downloading the file correctly (maybe it's not downloading it to the right location?).

File: api/css/full_scores_base.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (camelwater) [reply api/css/full_scores_base.css:17]: I think it's better though.

File: commands.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (BadWolf1023) [reply commands.py:147]: I believe it requires a Chrome driver.

**Pull request merged**